src/stdlib_terminal_colors_to_string.f90

```diff
@@ -0,0 +1,40 @@
+! SPDX-Identifier: MIT
+
+!> Implementation of the conversion to enumerator and identifier types to strings
+submodule (stdlib_terminal_colors) stdlib_terminal_colors_to_string
+    implicit none
+
+    character, parameter :: chars(0:9) = &
+        ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"]
+
+contains
+
+    !> Transform a color code into an actual ANSI escape sequence
+    pure module function to_string_ansi_code(code) result(str)
+        !> Color code to be used
+        type(ansi_code), intent(in) :: code
+        !> ANSI escape sequence representing the color code
+        character(len=:), allocatable :: str
+
+        if (anycolor(code)) then
+            str = achar(27) // "[0"  ! Always reset the style
```

Comment:
Maybe introduce a named parameter for `achar(27)` just to make it clear it's the ascii escape code:
```
character(len=1), parameter :: esc = achar(27)
```

Reply:
```suggestion
            str = esc // "[0"  ! Always reset the style
```